src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/PolicyFunctions.kt

```diff
@@ -10,28 +10,6 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.ConditionCh
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.ConditionChangeType.REPLACED
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.ConditionChangeType.TEXT_CHANGE
 
-fun <T : ILicenceCondition> removedConditions(current: List<T>, other: List<T>): List<T> =
-  current.filter { pssElement ->
-    !other.any { pss -> pss.code == pssElement.code }
-  }
-
-fun <T : ILicenceCondition> addedConditions(current: List<T>, other: List<T>): List<T> =
-  other.filter { pssElement ->
-    !current.any { pss -> pss.code == pssElement.code }
-  }
-
-fun <T : ILicenceCondition> amendedConditions(
```

Comment:
I assume these aren't shared methods with the licence change endpoint that we do use as a part of the inflight stuff?

Reply:
no, we'd get compilation failures if they were used elsewhere